### DISCUSSION TITLE
`wgpuTextureReference` Purpose

### DISCUSSION THREAD

**Discussion by jimbuck** [amerkoleci/Alimer.Bindings.WebGPU#4] [Announcements]
Hey, this is a great library for using WebGPU! I'm playing around with graphics programming and you're library looks the best when it comes to WebGPU bindings. In your samples I couldn't find any details on what the `wgpuTextureViewReference(targetView);` calls are doing. Do you have any links to documentation on what the `*Reference` methods do? Thanks in advanced!

**Comment by amerkoleci** [comment]
Hi,
Please refer to WebGPU documentation for this kind of questions, anyway wgpuTextureViewReference as all Reference methods increase reference count of the WebGPU object, while *Release releases reference count.

Cheers
